openedx/core/djangoapps/bookmarks/tests/test_tasks.py

```diff
@@ -103,11 +103,11 @@ def setUp(self):
         }
 
     @ddt.data(
-        (ModuleStoreEnum.Type.mongo, 2, 2, 3),
-        (ModuleStoreEnum.Type.mongo, 4, 2, 3),
-        (ModuleStoreEnum.Type.mongo, 2, 3, 4),
-        (ModuleStoreEnum.Type.mongo, 4, 3, 4),
-        (ModuleStoreEnum.Type.mongo, 2, 4, 5),
+        (ModuleStoreEnum.Type.mongo, 2, 2, 4),
+        (ModuleStoreEnum.Type.mongo, 4, 2, 4),
+        (ModuleStoreEnum.Type.mongo, 2, 3, 5),
+        (ModuleStoreEnum.Type.mongo, 4, 3, 5),
+        (ModuleStoreEnum.Type.mongo, 2, 4, 6),
```

Comment:
**Note to reviewers:** The query counts here went up by 1 only because the cache is now cleared before measuring the mongo calls.  Otherwise, it was resulting in discrepancies between CMS and LMS runtimes.